.github/workflows/unittest.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           esp_idf_version: v5.5
           target: esp32s3
-          command: GITHUB_ACTIONS="true" idf.py build
+          command: idf.py add-dependency "espressif/mdns^1.8.0" ; GITHUB_ACTIONS="true" idf.py build
```

Comment:
Why is this needed? The other dependencies are picked up automatically?

Reply:
Apparently that's what you have to do. Unsure why it's different, seems it's not part of the normal component repo? See https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/protocols/mdns.html